cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -1315,54 +1404,51 @@ func (s *initramfsMountsSuite) TestUnlockEncryptedPartition(c *C) {
 	tpm, err := tpm2.NewTPMContext(tcti)
```

Comment:
do we need this test now that we have the other one if we skip mocking this function?

Reply:
Testing this internal function is unnecessary now that we mock the secboot call directly.